docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,144 @@
+---
+
+## Webhooks
+
+In this guide, we will simplify how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data real time.  Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
+
+### What is a webhook?
+The main drivers of webhooks implementation are efficiency and observability.  Webhooks streamline collecting data between two connected services. 
+
+Like an API, webhooks are a mechanism for a client to recieve data. The main difference between webhooks and APIs is that a webhook functions as an inverted API.  In essence, the architecture of a webhook is composed of the following parts: 
+- You create and provide a client endpoint
+- You configure which type of event messages to listen to
+- The server broadcasts a message (event) to the client endpoint 
+- You verify the payload from the event message and send a 2xx back to the server
+ to trigger a task
+
+In this way, webhooks provide a real time connection between two services.  While APIs are useful for obtaining data on a scheduled basis, they are not efficient for handling real time requests when there is a high volume of activity.
+
+
+### Use Case
+In the case of Liveblocks, many of our users leverage our [rooms api](https://development.liveblocks.io/docs/api-reference/rest-api-endpoints)  to save data to their internal databases.  Prior to webhooks, you would need to set up a cron job to poll the API, and there was no ability implement real time updates. With webhooks, you can now subscribe to the events you are interested in, and be alerted of the change when it happens. 
+
+Under the hood, we have chosen to use [Svix](https://docs.svix.com/) as a webhooks provider- Svix makes it straightforward and intuitive to enable webhooks from inside of the Account Dashboard. They also provide additional measures to assure the that the webhook remains secure through verification. 
+
+## Messages
+For each activity that occurs under the defined list of events within the Event Catalog a message is generated.  Messages contain the metadata for each event which you may choose to subscribe to.  They also contain  event data and a alpha-numeric identifier.
+
+## Events
+Messages sent through Svix each have an `eventType` property which is used to identify and filter what is being sent to the client.  Each endpoint you provide in the webhooks dashboard listens to all events by default, but can be easily configured to only listen to a subset by updating the Message Filtering section.  As a new user, you may opt to listen to all events, and later evaluate which ones make the most sense to retain.  Alternatively, you can use the Event Catalog provided as way to determine which events make the most sense.  
+The Event Catalog provides a list of events available for subscription, along with their schema.
+
```

Comment:
I need to run a local instance to share a screen shot here for the event catalog. It is broken in dev.